pipelines/services/syncstatus.go

```diff
@@ -29,16 +30,14 @@ func (ss *SyncStatusService) GetSyncPipelineResourceStatus(repoName, branch stri
 	// Fetch resource ID
 	res, resourceErr := GetPipelineResource(ss.client, ss.GetUrl(), repoName, ss.getHttpDetails())
 	if resourceErr != nil {
-		log.Error("Unable to fetch resourceID for: ", repoName)
-		return []PipelineSyncStatus{}, resourceErr
+		return []PipelineSyncStatus{}, fmt.Errorf("%w; Second error", resourceErr)
```

Comment:
With the above change, you lost the daat you wanted to add to the error. The following part will not presented - 
```
Unable to fetch resourceID for: ", repoName
```
You can resolve this as follows - 
```
return []PipelineSyncStatus{}, fmt.Errorf("unable to fetch resourceID for '%s'. Second error: %s", repoName, resourceErr.Error())
```

Reply:
yeah this was a mistake i didnt push a commit